test/plugin/test_in_syslog.rb

```diff
@@ -59,14 +59,16 @@ def test_time_format
         {'msg' => '<6>Dec 11 00:00:00 localhost logger: foo', 'expected' => Fluent::EventTime.from_time(Time.strptime('Dec 11 00:00:00', '%b %d %H:%M:%S'))},
         {'msg' => '<6>Dec  1 00:00:00 localhost logger: foo', 'expected' => Fluent::EventTime.from_time(Time.strptime('Dec  1 00:00:00', '%b  %d %H:%M:%S'))},
       ]
-
-      d.run do
+      d.end_if do
+        d.emit_count >= 2
+      end
```

Comment:
Current, test driver implementation does not wait when block given.
So, in this case, `d.events` always returns empty events because `u.send` is heavy operation.